lib/tag_fetcher.rb

```diff
@@ -0,0 +1,29 @@
+require "gds_api/content_api"
+
+class TagFetcher
+
+  def initialize(manual)
+    @manual = manual
+  end
+
+  def fetch_tags
+    if artefact
+      artefact.tags
+    else
+      []
+    end
+  end
+
+private
+
+  attr_reader :manual
+
+  def artefact
+    content_api.artefact(manual.slug)
+  end
+
+  def content_api
+    @content_api ||= GdsApi::ContentApi.new(Plek.new.find("contentapi"))
+  end
+
+end
```

Comment:
This `TagFetcher` is quite specific to this app, so maybe it would live in `app/lib` instead?